src/config.js

```diff
@@ -1,7 +1,16 @@
 const config = {
 	wikiWhoUrl: 'https://www.wikiwho.net/',
 	// Set the output type between the extension and the gadget
-	outputEnvironment: ''
+	outputEnvironment: '',
+	// Elements that we know for sure are never tokenized and should be grayed out.
+	untokenizedElements: [
+		'.citation',
+		'.IPA',
+		'.mw-cite-backlink',
+		'.mw-editsection',
+		'.reference',
```

Comment:
This greys-out footnote superscript links, which do work. Is this by design?

Reply:
Yeah I wasn't sure... I grayed them out because they aren't tokenized (as in the `<ref>` tags themselves), but as you say the links work so no need to pretend they don't.